_includes/menu-item-ids/core-menu-items.md

```diff
@@ -12,6 +12,7 @@
 | `fontformats`        | Format                                                                                                   | Dropdown list with font families to apply to selection.                        |
 | `fontsizes`          | Format                                                                                                   | Dropdown list with font sizes to apply to selection.                           |
 | `italic`             | Format                                                                                                   | Applies italic format to current selection.                                    |
+| `language`           | Format                                                                                                   | Dropdown list with languages to apply to selection. {{site.requires_5_9v}}     |
```

Comment:
- [ ] Find a way to explain here (and in the matching toolbar button) that this is only available when the setting is specified (or at least link to the setting maybe)

Reply:
```suggestion
| `language`           | Format                                                                                                   | Dropdown list with languages to apply to selection. This item requires the [`content_langs` option]({{site.baseurl}}/configure/content-appearance/#content_langs).<br />{{site.requires_5_9v}}     |
```